pkg/reconciler/knativeeventing/common/replicasenvvarstransform.go

```diff
@@ -19,10 +19,17 @@ package common
 import (
 	mf "github.com/manifestival/manifestival"
 	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/client-go/kubernetes/scheme"
+	"knative.dev/pkg/system"
+)
+
+var (
+	envVarNames = []string{system.NamespaceEnvKey, "K_METRICS_CONFIG", "K_LOGGING_CONFIG", "K_LEADER_ELECTION_CONFIG",
```

Comment:
make this a `sets.String` to simplify the functions below (you won't need a find function anymore, as a set has a `Has` method)